packages/db/test/utils.js

```diff
@@ -21,6 +23,30 @@ export async function createUser (dbClient, options = {}) {
   return dbClient.getUser(issuer)
 }
 
+/**
+ * @param {import('../index').DBClient & {_client: import('@supabase/postgrest-js').PostgrestClient }} dbClient
+ * @param {number} userId
+ * @param {Object} [options]
+ * @param {string} [options.tag]
+ * @param {string} [options.value]
+ * @param {string} [options.reason]
+ */
+export async function createUserTag (dbClient, userId, options = {}) {
+  const { data } = await dbClient._client
+    .from('user_tag')
+    .upsert({
+      user_id: userId,
+      tag: options.tag || '',
+      value: options.value || '',
+      reason: options.reason || 'test reason'
+    })
```

Comment:
In order to be inline with the way the `user_tags` table is intended to work, this should really always do an insert, and set `deleted_at` on the previously-active row.

Reply:
Actually, we could also move this function into `packages/db/index.js` and then it would achieve this for us at the same time: #1195

Reply:
Have moved to the `db` package, now that its needed elsewhere.